### ISSUE TITLE
Could not load Microsoft.VisualStudio.ComponentModelHost

### ISSUE THREAD

**Issue by TimothyDJewell** [RandomEngy/UnitTestBoilerplateGenerator#3]
### Installed product versions
- Visual Studio: [2017 Professional]
- This extension: [2.5.4]

### Description
When opening VS, I get a warning popup from VS: "The 'CreateUnitTestBoilerplateCommandPackage' package did not load correctly."   It suggests opening ActivityLog.xml to find details.  When looking up those details, I find 3 relevant entries:
```xml
  <entry>
    <record>778</record>
    <time>2021/04/05 19:55:40.875</time>
    <type>Information</type>
    <source>VisualStudio</source>
    <description>Begin package load [CreateUnitTestBoilerplateCommandPackage]</description>
    <guid>{73CD6DCD-1791-4633-ACF4-889C1A09CA55}</guid>
  </entry>
  <entry>
    <record>779</record>
    <time>2021/04/05 19:55:40.953</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [CreateUnitTestBoilerplateCommandPackage]Source: &apos;UnitTestBoilerplate&apos; Description: Could not load file or assembly &apos;Microsoft.VisualStudio.ComponentModelHost, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;System.IO.FileNotFoundException: Could not load file or assembly &apos;Microsoft.VisualStudio.ComponentModelHost, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;File name: &apos;Microsoft.VisualStudio.ComponentModelHost, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos;&#x000D;&#x000A;   at UnitTestBoilerplate.CreateUnitTestBoilerplateCommandPackage.&lt;InitializeAsync&gt;d__2.MoveNext()&#x000D;&#x000A;   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine&amp; stateMachine)&#x000D;&#x000A;   at UnitTestBoilerplate.CreateUnitTestBoilerplateCommandPackage.InitializeAsync(CancellationToken cancellationToken, IProgress`1 progress)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.AsyncPackage.&lt;&gt;c__DisplayClass16_0.&lt;&lt;Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize&gt;b__1&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.GetResult()&#x000D;&#x000A;&#x000D;&#x000A;WRN: Assembly binding logging is turned OFF.&#x000D;&#x000A;To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.&#x000D;&#x000A;Note: There is some performance penalty associated with assembly bind failure logging.&#x000D;&#x000A;To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].&#x000D;&#x000A;</description>
    <guid>{73CD6DCD-1791-4633-ACF4-889C1A09CA55}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>
  <entry>
    <record>780</record>
    <time>2021/04/05 19:55:40.969</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [CreateUnitTestBoilerplateCommandPackage]</description>
    <guid>{73CD6DCD-1791-4633-ACF4-889C1A09CA55}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>
```
The description for that error is:
> SetSite failed for package [CreateUnitTestBoilerplateCommandPackage]Source: 'UnitTestBoilerplate' Description: Could not load file or assembly 'Microsoft.VisualStudio.ComponentModelHost, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.VisualStudio.ComponentModelHost, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
File name: 'Microsoft.VisualStudio.ComponentModelHost, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at UnitTestBoilerplate.CreateUnitTestBoilerplateCommandPackage.<InitializeAsync>d__2.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine)
   at UnitTestBoilerplate.CreateUnitTestBoilerplateCommandPackage.InitializeAsync(CancellationToken cancellationToken, IProgress`1 progress)
   at Microsoft.VisualStudio.Shell.AsyncPackage.<>c__DisplayClass16_0.<<Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)
   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)
   at Microsoft.VisualStudio.Services.VsTask.GetResult()
> 
> WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

### Steps to recreate
1. Open VS.
2. See error.

### Current behavior
Extension does not load/cannot be used.

### Expected behavior
Extension loads without issue.

**Comment by TimothyDJewell** [comment]
This began last week, I believe.  I have tried uninstalling the extension and reinstalling to get it to work, but it still did not load.  Looking at recent commits, it seems likely related to bb7d0cf699a (especially since the most recent update date for me was 3/31, though I don't understand enough about extensions or dependency loading to accurately diagnose.

I don't see a way to download a previous version to verify that this is the issue or to let me use the extension as it was.  (It's one of my favorites, and I frequently encourage my teammates to use it to save time.)

**Comment by RandomEngy** [comment]
I was trying to build but it was giving me an error about Microsoft.VisualStudio.ComponentModelHost, so I upgraded that DLL and that resolved it. But also apparently broke it for VS 2017.

If you use Visual Studio 2019 will probably load without error. I'll also look into getting it working again for 2017.

**Comment by RandomEngy** [comment]
Should be fixed in v2.5.5, which I just released. Tested on VS 2017.
